src/components/MultiActionButton/__tests__/MultiActionButton.spec.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import { MultiActionButton } from '../MultiActionButton';
+import { Button } from '../../Buttons';
+import { ListItem } from '../../List';
+
+describe('MultiActionButton', () => {
+    const mockOnChange = jest.fn();
+
+    it('should render correctly closed', () => {
+        const wrapper = mount(
+            <MultiActionButton
+                button={<Button context="brand">Click me!</Button>}
+                onChange={mockOnChange}
+                placement="top-start"
+            >
+                <ListItem key="disabled-key" disabled>
+                    Disabled
+                </ListItem>
+                <ListItem key="first-key" value="first-value">
+                    With value
+                </ListItem>
+            </MultiActionButton>
+        );
```

Comment:
Wouldn't it be easier to mount the component in a `beforeEach` block, and only mount it inside `it` when it is different from the ususal?

Reply:
Agree! It is more obvious and you don't need to check if something was changed in component for current test